packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartBankForm/CounterpartBankForm.test.tsx

```diff
@@ -207,56 +203,59 @@ describe('CounterpartBankForm', () => {
       const sortCode = '[create] Sort code';
       const countrySelectName = /country/i;
 
-      await waitFor(
-        () =>
-          expect(
-            screen.findByRole('button', { name: countrySelectName })
-          ).resolves.not.toBeDisabled(),
-        { timeout: 5_000 }
+      await waitFor(() =>
+        expect(
+          screen.getByRole('button', { name: countrySelectName })
+        ).not.toBeDisabled()
       );
 
       triggerClickOnSelectOption(countrySelectName, 'United Kingdom');
       triggerClickOnAutocompleteOption(/currency/i, /Armenian/i);
-      triggerChangeInput(/account name/i, accountName);
-      triggerChangeInput(/iban/i, iban);
-      triggerChangeInput(/bic/i, bic);
-      triggerChangeInput(/account number/i, accountNumber);
-      triggerChangeInput(/sort code/i, sortCode);
+
```

Comment:
I don't get the reasons for these changes. The test is passed with `triggerChangeInput` as well.